src/validation.cpp

```diff
@@ -77,6 +77,14 @@ static constexpr std::chrono::hours DATABASE_WRITE_INTERVAL{1};
 static constexpr std::chrono::hours DATABASE_FLUSH_INTERVAL{24};
 /** Maximum age of our tip for us to be considered current for fee estimation */
 static constexpr std::chrono::hours MAX_FEE_ESTIMATION_TIP_AGE{3};
+extern const std::vector<std::string> CHECKLEVEL_DOC = {
```

Comment:
```suggestion
const std::vector<std::string> CHECKLEVEL_DOC{
```

nit: no need for the `=`

Reply:
addressed